diff --git a/arch/arm64/boot/dts/rockchip/rk3576-recomputer-rk3576-devkit.dts b/arch/arm64/boot/dts/rockchip/rk3576-recomputer-rk3576-devkit.dts
index 28afce0..76360dc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3576-recomputer-rk3576-devkit.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3576-recomputer-rk3576-devkit.dts
@@ -42,7 +42,7 @@
 		status = "okay";
 		compatible = "pwm-fan";
 		#cooling-cells = <2>;
-		cooling-levels = <0 50 100 150 200 255>;
+		cooling-levels = <25 45 60 75 100 130 170 210 255>;
 		fan-supply = <&vcc_sys>;
 		pwms = <&pwm0_2ch_0 0 60000 0>;
 
@@ -53,10 +53,14 @@
 		pinctrl-0 = <&fan_tach>;
 
 		rockchip,temp-trips = <
-            20000    1
-            25000    2
-            30000    3
-            35000    5
+            38000    1
+            44000    2
+            50000    3
+            56000    4
+            62000    5
+            68000    6
+            74000    7
+            80000    8
         >;
 	};
 
@@ -758,7 +762,7 @@
 		interrupt-controller;
 		gpio-line-names =
 		"MIPI_DPHY_CSI0_IO1"	,"USB_USB30_PWREN_H",
-		"USB_HOST_PWREN_H"	,"PCIe_SX1261_RST_3V3",
+		"USB_HOST_PWREN_H"	,"BRD_ID_WP",
 		"LTE_RESET"	,"P05",
 		"P06"	,"LoRaWan_sx1262_CS",
 		"USER_LED_R"	,"USER_LED_B",
diff --git a/arch/arm64/boot/dts/rockchip/rk3588-recomputer-rk3588-devkit.dts b/arch/arm64/boot/dts/rockchip/rk3588-recomputer-rk3588-devkit.dts
index 4862cc7..34a2ffd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-recomputer-rk3588-devkit.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-recomputer-rk3588-devkit.dts
@@ -205,7 +205,7 @@
 		compatible = "pwm-fan";
 		#cooling-cells = <2>;
 		pwms = <&pwm2 0 50000 0>;
-		cooling-levels = <0 50 100 150 200 255>;
+		cooling-levels = <25 45 60 75 100 130 170 210 255>;
 		// cooling-levels = <0 255>;
 
         interrupt-parent = <&gpio3>;
@@ -219,11 +219,14 @@
 		// >;
 
 		rockchip,temp-trips = <
-			40000   1
-			45000   2
+			38000   1
+			44000   2
 			50000   3
-			55000   4
-			60000   5
+			56000   4
+			62000   5
+			68000   6
+			74000   7
+			80000   8
 		>;
  
 		status = "okay";
@@ -354,7 +357,7 @@
 	no-sdio;
 	no-sd;
 	non-removable;
-	max-frequency = <150000000>;
+	max-frequency = <100000000>;
 	mmc-hs200-1_8v;
 	status = "okay";
 };
@@ -378,7 +381,7 @@
 
 
 &sdio {
-	max-frequency = <150000000>;
+	max-frequency = <100000000>;
 	supports-sdio;
 	bus-width = <4>;
 	disable-wp;
